packages/@aws-cdk/aws-amplify-alpha/lib/domain.ts

```diff
@@ -131,6 +131,13 @@ export class Domain extends Resource {
     this.subDomains = props.subDomains || [];
 
     const domainName = props.domainName || id;
+    if (domainName.length > 255) {
+      throw new Error(`Domain name must be 255 characters or less, got: ${domainName.length}`);
```

Comment:
```suggestion
      throw new Error(`Domain name must be 255 characters or less, got: ${domainName.length} characters.`);
```

I think it would be more helpful to include "characters" in the error message.